gdp/small_batch/gdp_small_batch.py

```diff
@@ -215,10 +224,13 @@ def units(m, j):
     def lim(m, j):
         """
         Only one choice for parallel units is feasible.
+        Equation:
+            Sum(m.Y[k,j] for j in m.j) == 1, for k in m.k
 
         Args:
             m (pyomo.ConcreteModel): small batch GDP model
             j (str): stage
+            k (int): number of parallel units
```

Comment:
There is no k argument in this function

Reply:
You are not reading my comments. This function has no parameter k, so why do you add the documentation saying so?